doc/exports.md

```diff
@@ -0,0 +1,44 @@
+# Exports
+
+## Spending Breakdown
+A report primarily aimed at BEIS finance needs.
+
+- Includes all __Actuals spend__ at the time the export is created
+- Includes all __Refunds__ at the time the export is created
+- Actual spend includes adjustments for the same financial quarter and of `type`
+  `Actual`
+- Refunds includes adjustments for the same financial quarter and of `type`
+  `Refund`
+- Actuals spend, Refunds and Adjustment share the same `Transaction` base class
+  and database table
+- Includes a net total, the result of summing the Actual spend and Refund totals
+- Includes all `Forecast` for financial quarters not already included with Actual
+  spend and refunds at the time the export is created
+- Forecasts are already stored as a total for the activity and financial quarter
+
+### Example calculation
+
+```
+Q1 2021-2022 values
+---------------------------------
+Actual spend:              10,000
+Actual spend:               5,000
+Adjustment (type: Actual): -1,000
+Refund:                    -5,000
+Adjustment (type: Refund):  1,000
+Adjustment (type: Actual):   -500
```

Comment:
and we could then lose this line as we already have a negative Actual Adjustment in the example